### ISSUE TITLE
Error -  "Callback was already called."

### ISSUE THREAD

**Issue by dgresko** [tettusud/merge-jsons-webpack-plugin#27]
When I try to compile two files separately, just as you illustrate in the README, it fails with the following error:
![image](https://user-images.githubusercontent.com/5972921/35346662-d7c187c6-0100-11e8-836c-5b9bee4413d3.png)

```
    plugins: [
        new MergeJsonWebpackPlugin({
            'output': {
                'groupBy': [
                    {
                        'pattern': '{./src/**/translations/en.json, ./src/translations/en.json}',
                        'fileName': 'assets/translations/en.json'
                    },
                    {
                        'pattern': '{./src/**/translations/fr.json, ./src/translations/fr.json}',
                        'fileName':'assets/translations/fr.json'
                    }
                ]
            }
        })
    ]
```

It works for _either_ the fr.json or the en.json files but not when both are in there.
Angular version: 5.1.2
Merge-json Version 1.0.12

**Comment by tettusud** [comment]
@dgresko let me start looking into it.Thank you

**Comment by tettusud** [comment]
@dgresko I released new version 1.0.13 could you test with that. I am not able to test your issue on my side. if the issue still persists please let me know.

**Comment by dgresko** [comment]
Yes! This is resolved. Nice work. Thank you.
